From 3028c78c8c2dcc644003060e320fd10ede725ad0 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 03 Jun 2020 23:51:12 +0300
Subject: [PATCH] [SW-0] i2cslave: fix the "processed" message

The I2C slave "read request processed" message lacks the EOL and results
in overwriting the previous message.
Add the missing EOL to the message.

Change-Id: I1f6c4520381abae21d93d2ee342814bceedacf49
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/module_i2c_slave.c b/common/module_i2c_slave.c
index 19ad37c..d5c0ca6 100644
--- a/common/module_i2c_slave.c
+++ b/common/module_i2c_slave.c
@@ -240,6 +240,6 @@
 		ret = i2cslave_handle_read_req();
 		read_bytes_count++;
 		regmap_writel(ret, mmPSOC_I2C_S_IC_DATA_CMD);
-		preboot_puts("I2C SLAVE: read request processed");
+		preboot_puts("I2C SLAVE: read request processed\n");
 	}
 }
